src/keypair.rs

```diff
@@ -354,6 +375,20 @@ mod tests {
         sign_test_keypair(&Keypair::TPM(keypair));
     }
 
+    #[cfg(feature = "nova-tz")]
+    #[test]
+    fn sign_tz() {
+        fs::write(
+            "/tmp/rsa_key_blob",
```

Comment:
I suggest using https://crates.io/crates/tempfile as a dev-dependency to avoid file location/collision  issues and using it here